examples/bluetooth/ble_advertising.py

```diff
@@ -50,6 +55,8 @@ def _append(adv_type, value):
     if appearance:
         _append(_ADV_TYPE_APPEARANCE, struct.pack("<h", appearance))
 
+    if len(payload) > _ADV_MAX_PAYLOAD:
+        raise ValueError(f"ADV payload too large. Max {_ADV_MAX_PAYLOAD} got {len(payload)}")
```

Comment:
I think a simple message would suffice, eg "advertising payload too large" (this follows the general philosophy of MicroPython to keep things minimal where appropriate).